.github/workflows/codeql-analysis.yml

```diff
@@ -13,7 +13,7 @@ name: "CodeQL"
 
 on:
   push:
-    branches: [ master, * ]
+    branches: [ master ]
```

Comment:
Are the cloud triggers configured okay?

https://screenshot.googleplex.com/8cRTdarbk7eku72
https://screenshot.googleplex.com/52poe9PrBEhuuRu

Reply:
yes they were working well on this pr. i had added the branch protection before adding cloudql, and i have since removed it since it was a bit too aggressive (e.g. couldn't create dependabot fixes). we can discuss at our standup what to do instead.